tests/transformers/tests/test_modeling_common.py

```diff
@@ -1883,8 +1883,8 @@ def test_multi_gpu_data_parallel_forward(self):
 
         # some params shouldn't be scattered by nn.DataParallel
         # so just remove them if they are present.
-        blacklist_non_batched_params = ["head_mask", "decoder_head_mask", "cross_attn_head_mask"]
-        for k in blacklist_non_batched_params:
+        blocklist_non_batched_params = ["head_mask", "decoder_head_mask", "cross_attn_head_mask"]
```

Comment:
why do we make this change, is it related?

Reply:
Not related. just another offending code